src/content/docs/cypher/query-clauses/call.md

```diff
@@ -27,20 +27,22 @@ The following tables lists the built-in schema functions you can use with the `C
 | `property id` | Internal identifier of the property within table | INT64 |
 | `name` | name of the property | STRING |
 | `type` | data type of the property | STRING |
+| `default expression` | default expression of the property | STRING |
 | `primary key` | if property is primary key | BOOLEAN |
 
 ```cypher
 CALL TABLE_INFO('User') RETURN *;
 ```
 Output:
 ```
----------------------------------------------
-| property id | name | type   | primary key |
----------------------------------------------
-| 0           | name | STRING | True        |
----------------------------------------------
-| 1           | age  | INT64  | False       |
----------------------------------------------
+-----------------------------------------------------------------
```

Comment:
The new version of the CLI generates prettier tables - could you please rerun the example in the latest CLI and copy that output here instead of reusing the existing one?